emcc.py

```diff
@@ -1401,6 +1401,11 @@ def default_setting(name, new_default):
       # memory init file is not supported with side modules, must be executable synchronously (for dlopen)
       options.memory_init_file = False
 
+    # If we are including the ensure JS library then we know for sure we will by definition
```

Comment:
"If we are including the ensure JS library"? -> "If we are including the full JS library"?